samcli/local/apigw/local_apigw_service.py

```diff
@@ -358,8 +365,9 @@ def _parse_lambda_output(lambda_output, binary_types, flask_request):
         headers = LocalApigwService._merge_response_headers(
             json_output.get("headers") or {}, json_output.get("multiValueHeaders") or {}
         )
+
         body = json_output.get("body")
-        if body is None:
+        if not body:
```

Comment:
Why this difference? I'd actually like a more explicit `if` condition (if `None` is insufficient, let's enumerate the cases here).